package.json

```diff
@@ -35,6 +35,6 @@
 	},
 	"devDependencies": {
 		"@tc39/ecma262-biblio": "^2.1.2682",
```

Comment:
let's update the biblio too

Reply:
That causes the build to fail because the `Type` AO was removed in v2.1.2773.